### ISSUE TITLE
OpenPGP responses do not get decrypted

### ISSUE THREAD

**Issue by almereyda** [thunderbird-conversations/thunderbird-conversations#1634]
* **Thunderbird version:** 78.7.1
* **Thunderbird Conversation version:** 3.2.17
* **Operating System:** Fedora 33

**What did you do?**

- Open an unecrypted message
- Respond with encryption enabled and the message saved to Sent.
- Reopen the conversation

**What did you expect to happen?**

- All messages are readable.

**What actually happened?**

- The unencrypted message is readable.
- The encrypted message is displayed empty.
- Opening the encrypted message in classical view displays it decrypted.
- Opening the conversation from the encrypted message as start, it displays with the tag 'decrypted' as expected.

**Comment by Standard8** [comment]
I've a feeling this has always been an issue, and is down to how Thunderbird decrypts messages currently. As I said in the other issue, fixing this is probably significant work, but maybe it is about time to tackle it.

**Comment by almereyda** [comment]
Yes, this is a very new terrain. Also the side-effects that have been created with the OpenPGP implementation are

<details>

<summary>not all the nicest</summary>

* [Storage of PGP passphrase | Thunderbird Support Forum | Mozilla Support](https://support.mozilla.org/en-US/questions/1304363)
  * [Thunderbird:OpenPGP:Smartcards - MozillaWiki](https://wiki.mozilla.org/Thunderbird:OpenPGP:Smartcards)
  * **[1566458 - Optionally protect the startup of Thunderbird using the Master Password](https://bugzilla.mozilla.org/show_bug.cgi?id=1566458)** comment `#20` nails it
    * [1681262 - encrypted emails are automatically decrypted without a password provided](https://bugzilla.mozilla.org/show_bug.cgi?id=1681262)
    * [1680300 - After closing master password prompt, it shows account and all messages without any protection](https://bugzilla.mozilla.org/show_bug.cgi?id=1680300)
    * [1688470 - Locking pgp key/stopping pgp](https://bugzilla.mozilla.org/show_bug.cgi?id=1688470)

</details>

**Comment by Standard8** [comment]
Whilst I appreciate some users take issue with Thunderbird's OpenPGP implementation, the only parts Conversations needs to be concerned about is the actual integration points, anything else is off-topic.

**Comment by sma-ops** [comment]
Is there any progress on this one? Without decrypting the messages in a conversation, the add-on is basically unusable for me.

Confirming the issue for:
Thunderbird version: 102.5
Thunderbird Conversations version: 4.0.25
OS: Ubuntu 22.04
